### ISSUE TITLE
Wie finde ich ID und Name der Region

### ISSUE THREAD

**Issue by blurap** [redlukas/wetter-alarm#77]
Gute Integration. Können sie mir sagen, wie ich ID und Name der Region rausfinde? Vielen Dank schon jetzt für die Hilfe.

**Comment by sinabuddy** [comment]
Perhaps from here? Bsp AG:
https://github.com/redlukas/wetter-alarm/blob/master/POIs/by-canton/AG.md
 I don't now...

**Comment by RaphB68009** [comment]
Ich habe die ID und Regionamen aus dem Link probiere. 
Die Integration wird dadurch auch gestartet... Allerdings erstellt es mir keinerlei Entitäten

**Comment by okaufmann** [comment]
@sinabuddy @RaphB68009 @blurap Bitte ladet die neuste Version herunter und versucht es erneut.

**Comment by RaphB68009** [comment]
Hab diese gerade heruntergeladen und aktualisiert. 
Integration funktioniert aber immer noch keine Entitäten.

**Comment by blurap** [comment]
Danke für die Rückmeldung.

Habe ebenfalls die Integration gelöscht und neu heruntergeladen und installiert. Sie scheint zu funktionieren, liefert aber keine Entitäten. 

Als ID habe ich eine sechsstellige Nummer POI ID aus oben angegebener Liste verwendet (z.B. 140770).

**Comment by redlukas** [comment]
Ja, ich muss erst noch #83 reviewen und mergen. dann kannst du es nochmal versuchen.

**Comment by okaufmann** [comment]
Bitte entschuldigt die Verwirrung, ich war zu schnell mit dem Kommentar 🙈

**Comment by RaphB68009** [comment]
kein Ding. 
Oder ich war einfach zu schnell im gerade ausprobieren ^^
